Traceback (most recent call last):
  File "/home/hellorahat/Github/NavMaze/CrossCompute/Phase1/Iteration3/run.py", line 181, in <module>
    with open(completeName, 'r') as file:
FileNotFoundError: [Errno 2] No such file or directory: '/home/hellorahat/Github/NavMaze/CrossCompute/Phase1/Iteration3/batches/a/input/data.csv'
